distributions/kfdef/kfctl_openshift_v1.3.0_servicemesh.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: kubeflow
+  namespace: kubeflow
+spec:
+  applications:
+    - kustomizeConfig:
+        parameters:
+          - name: namespace
+            value: openshift-operators
+        repoRef:
+          name: odhmanifests
+          path: openshift-service-mesh/base
+      name: openshift-sm
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/openshift/openshift-scc/overlays/servicemesh
+      name: openshift-scc
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/cert-manager
+      name: cert-manager
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/admission-webhook/overlays/servicemesh
+      name: admission-webhook
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/openshift-service-mesh
+      name: istio-managed
+    - kustomizeConfig:
```

Comment:
Can we change the name from istio-managed to service-mesh or istio-service-mesh

Reply:
Updated to `istio-service-mesh`